app/src/lib/stores/pull-request-coordinator.ts

```diff
@@ -156,6 +156,33 @@ export class PullRequestCoordinator {
     }
   }
 
+  /**
+   * Get the last time a repository's pull requests were fetched
+   * from the GitHub API
+   *
+   * Since `PullRequestStore` stores these timestamps by
+   * GitHubRepository, we gather the timestamps for all
+   * **related repos** and return _the least recent one._
```

Comment:
Can we clarify the `related repos` part so that it's clear that there's only ever going to be one or two repositories? Related repositories could mean a lot of things

Additionally, could we say "the oldest" instead of "the least recent"?

```suggestion
   * the repository and its parent and return _oldest one._
```

Reply:
good point! i did a couple revisions on this